R/param_range_limits.R

```diff
@@ -0,0 +1,37 @@
+#' Limits for the range of predictions
+#'
+#' Range limits truncate model predictions to a specific range of values,
+#' typically to avoid extreme or unrealistic predictions.
+#'
+#' @inheritParams Laplace
+#' @examples
+#' lower_limit()
+#' upper_limit()
+#' @name range_limits
+#' @export
+
+#' @rdname range_limits
+#' @export
+lower_limit <- function(range = c(-Inf, Inf), trans = NULL) {
+  new_quant_param(
+    type = "double",
+    range = range,
+    inclusive = c(TRUE, FALSE),
```

Comment:
A second set of eyes on this for Wonk Prevention would be appreciated. We allow lower limits from [-Inf, Inf)—i.e. one can't set a lower limit of infinity.

Reply:
That sounds resonable to me